### ISSUE TITLE
Return HTTP 404 / 403 for missing data

### ISSUE THREAD

**Issue by evancharlton** [jossi87/climbing-web#24]
Currently, some of the endpoints return 500 status code for either a missing resource or invalid permissions. Neither of these are appropriate for HTTP 500, which should indicate an error on the server

Could we simplify this logic and return an HTTP 404 when an invalid resource is requested? We could also consider returning an HTTP 403 if the user doesn't have access to the resource, too.

**Comment by jossi87** [comment]
Agree, will fix this in the web service in the next couple of days
